compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/sink_into_core.mlir

```diff
@@ -0,0 +1,72 @@
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(iree-amdaie-sink-into-core)" %s | FileCheck %s
```

Comment:
We should add a test with AMDAIE DMA operations being used inside the cores as the pass has logic to handle that.